src/main/java/com/stripe/model/SubscriptionItem.java

```diff
@@ -1,164 +1,217 @@
    */
-  public UsageRecordSummaryCollection usageRecordSummaries(Map<String, Object> params,
-      RequestOptions options) throws StripeException {
-    String url = instanceUrl(SubscriptionItem.class, this.getId()) + "/usage_record_summaries";
-    return requestCollection(url, params, UsageRecordSummaryCollection.class, options);
+  public UsageRecordSummaryCollection usageRecordSummaries() throws StripeException {
+    return usageRecordSummaries((Map<String, Object>) null, (RequestOptions) null);
   }
-  // </editor-fold>
 
-  // <editor-fold desc="update">
   /**
-   * Update a subscription item.
+   * For the specified subscription item, returns a list of summary objects. Each object in the list
+   * provides usage information that’s been summarized from multiple usage records and over a
+   * subscription billing period (e.g., 15 usage records in the billing plan’s month of September).
+   *
+   * <p>The list is sorted in reverse-chronological order (newest first). The first list item
+   * represents the most current usage period that hasn’t ended yet. Since new usage records can
+   * still be added, the returned summary information for the subscription item’s ID should be seen
+   * as unstable until the subscription billing period ends.
    */
-  public SubscriptionItem update(Map<String, Object> params) throws StripeException {
-    return update(params, null);
+  public UsageRecordSummaryCollection usageRecordSummaries(Map<String, Object> params)
+      throws StripeException {
+    return usageRecordSummaries(params, (RequestOptions) null);
   }
 
   /**
-   * Update a subscription item.
+   * For the specified subscription item, returns a list of summary objects. Each object in the list
+   * provides usage information that’s been summarized from multiple usage records and over a
+   * subscription billing period (e.g., 15 usage records in the billing plan’s month of September).
+   *
+   * <p>The list is sorted in reverse-chronological order (newest first). The first list item
+   * represents the most current usage period that hasn’t ended yet. Since new usage records can
+   * still be added, the returned summary information for the subscription item’s ID should be seen
+   * as unstable until the subscription billing period ends.
    */
-  public SubscriptionItem update(Map<String, Object> params, RequestOptions options)
-      throws StripeException {
-    return request(RequestMethod.POST, instanceUrl(SubscriptionItem.class, id), params,
-        SubscriptionItem.class, options);
+  public UsageRecordSummaryCollection usageRecordSummaries(
+      Map<String, Object> params, RequestOptions options) throws StripeException {
+    String url =
+        String.format(
+            "%s%s",
+            Stripe.getApiBase(),
+            String.format("/v1/subscription_items/%s/usage_record_summaries", this.getId()));
+    return requestCollection(url, params, UsageRecordSummaryCollection.class, options);
```

Comment:
8) Retrieve (custom): Instance method retrieving sub-resource from an instance, relying on instance ID attribute. 
Method name is the sub-resource instead of the standard retrieve.

Reply:
https://github.com/stripe/stripe-java/blob/2aa5db5c39dc6e91228507798083f373904cc8c0/src/main/java/com/stripe/model/SubscriptionItem.java#L189-L207